applications/plugins/SofaPython/CMakeLists.txt

```diff
@@ -141,13 +141,43 @@ set(PYTHON_FILES
     python/SofaPython/PythonAdvancedTimer.py
 )
 
-
 find_package(PythonLibs 2.7 REQUIRED)
 find_package(SofaGui REQUIRED)
 find_package(SofaUserInteraction REQUIRED)
 find_package(SofaMiscMapping REQUIRED)
 find_package(Eigen3 REQUIRED)
 
+sofa_find_package(SofaTest QUIET)
```

Comment:
find_package or sofa_find_package :)

Reply:
sofa_find_package to get the cmake variable SOFAPYTHON_HAVE_SOFATEST defined in the cmake.in, etc

Reply:
So all the other find_package in every CMakeLists.txt should become sofa_find_package ?